composer.json

```diff
@@ -14,14 +14,14 @@
   "require": {
     "php": ">=5.6",
     "psr/log": "^1.0",
-    "symfony/symfony": "^2.6|^3.0",
+    "symfony/symfony": "^2.7|^3.0",
     "doctrine/orm": "^2.3",
-    "treehouselabs/queue": "^v0.1.0"
+    "treehouselabs/queue": "dev-master"
```

Comment:
Wouldn't it be better to tag 0.2.0 as a separate release and require that instead? Requiring `dev-master` is always a bit tricky and should be a last resort solution IMHO.